third_party/terraform/resources/resource_logging_bucket_config.go

```diff
@@ -115,8 +120,90 @@ func resourceLoggingBucketConfigAcquire(iDFunc loggingBucketConfigIDFunc) func(*
 
 		d.SetId(id)
 
-		return resourceLoggingBucketConfigUpdate(d, meta)
+		log.Printf("[DEBUG] Fetching logging bucket config: %#v", d.Id())
+		url, err := replaceVars(d, config, fmt.Sprintf("{{LoggingBasePath}}%s", d.Id()))
+		if err != nil {
+			return err
+		}
+
+		res, err := sendRequest(config, "GET", "", url, userAgent, nil)
+		if res != nil {
+			log.Printf("[DEGUG] Acquired logging bucket config at %s", d.Id())
+			// Logging bucket already exists, update is called
+			return resourceLoggingBucketConfigUpdate(d, meta)
+		} else {
+			return resourceLoggingBucketConfigCreate(d, meta)
+		}
+	}
+}
+
+func resourceLoggingBucketConfigCreate(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	userAgent, err := generateUserAgentString(d, config.userAgent)
+	if err != nil {
+		return err
+	}
+
+	obj := make(map[string]interface{})
+	nameProp, err := expandLoggingBucketName(d.Get("name"), d, config)
```

Comment:
These expander functions aren't needed since they just pass the value of `d.Get` through and return nil for the error. It should be fine to just do something similar to 
```
obj[name] = d.Get("name")
```
for all of the fields and then remove the expanders. Generated resources have these all over the place just because it is the best way template many scenarios